src/styles/theme.css

```diff
@@ -20,6 +20,7 @@
     --primary-700: hsl(var(--primary), 35%);
     --primary-800: hsl(var(--primary), 25%);
     --primary-900: hsl(var(--primary), 15%);
+    --primary-blue: hsl(var(--primary), 10%);
```

Comment:
This is the original dark blue used for the title header - open to a better variable name if there are suggestions!